test/unit/assorted/optional_require.test.js

```diff
@@ -53,24 +53,17 @@ describe('optionalRequire', function () {
     });
   });
 
-  context('aws4', {
-    metadata: {
-      requires: {
-        mongodb: '>=4.4'
+  context('aws4', function () {
+    it('should error if not installed', function () {
```

Comment:
Thanks for removing the metadata stuff from the unit tests! Is this all of them? it should be safe to remove the `"ui"` setting from the root `.mocharc.json` if so, then future tests will error if metadata is accidentally reintroduced.

Reply:
I think so